backend/config/curation-api.yml

```diff
@@ -44,12 +44,14 @@ paths:
     get:
       summary: Fetch Collections metadata.
       description: Fetch all Collections metadata. Authorization required for private collections.
+      operationId: backend.corpora.lambdas.api.v1.collection.get_collections_curation
       security:
         - curatorAccessLenient: []
-      parameters:
-        - $ref: "#/components/parameters/query_visibility"
+        - {}
       tags:
         - collection
+      parameters:
+        - $ref: "#/components/parameters/query_visibility_default_null"
```

Comment:
I couldn't figure out how to override the default value of `"PUBLIC"` to make it null for `#/components/parameters/query_visibility`, so this is my solution